docker/Dockerfile

```diff
@@ -58,4 +56,7 @@ RUN addgroup --system --gid ${UID} ${USER} ; \
 
 USER ${USER}
 
+RUN composer global require --no-progress --no-scripts --no-plugins symfony/flex ; \
+    composer global config --no-plugins allow-plugins.symfony/flex true
```

Comment:
I actually think this isn't required. Same with the flex config in composer.json (just having flex installed globally enables the SYMFONY_REQUIRE)

Reply:
nm, this is required.